staging/knative/charts/eventing-sources/charts/github/values.yaml

```diff
@@ -0,0 +1,2 @@
+global:
+  serviceLabels: {}
```

Comment:
by looking at the configuration options, this doesn't look much as a chart. It looks more like a bunch of yaml files that could be installed with a kubectl create command without any customization